go.sum

```diff
@@ -389,6 +389,10 @@ github.com/hashicorp/yamux v0.1.1 h1:yrQxtgseBDrq9Y652vSRDvsKCJKOUD+GzTS4Y0Y8pvE
 github.com/hashicorp/yamux v0.1.1/go.mod h1:CtWFDAQgb7dxtzFs4tWbplKIe2jSi3+5vKbgIO0SLnQ=
 github.com/hewlettpackard/hpegl-metal-client v1.5.22 h1:EW19Adpfroi27jpYYjlNddx/TwGKnTVysd5Lpqoo/ls=
 github.com/hewlettpackard/hpegl-metal-client v1.5.22/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48 h1:eaHwFgw9BaQn7O7AMchZgOBxfOygrP7lXKoAHUmsWEc=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23-0.20240923072040-5b07002ffb48/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23 h1:EQuYg4qGpUUF11hNYqb3ekzUKgqIfQd2RNg1Y3SCGqc=
+github.com/hewlettpackard/hpegl-metal-client v1.5.23/go.mod h1:+BO4BfQXf3IWnsxvk2IdE27Ijrm1jDVYIgpebYh1L9M=
```

Comment:
Did you `go mod tidy`? I am surprised to see v1.5.22 still in `go.sum`.

Reply:
Updated.